src/utils/upload/processUpload.js

```diff
@@ -22,6 +22,16 @@ const putInS3 = async (projectUuid, loadedFileData, dispatch, sampleUuid, fileNa
   const signedUrlResponse = await fetchAPI(
     url,
     {
+      metadata,
+      progressCallback(progress) {
+        const percentProgress = Math.round((progress.loaded / progress.total) * 100);
+        dispatch(updateSampleFile(sampleUuid, fileName, {
+          upload: {
+            status: UploadStatus.UPLOADING,
+            progress: percentProgress ?? 0,
+          },
+        }));
+      },
```

Comment:
how this came about is a mystery to me -- it is not currently in master, I don't remember adding it in

Reply:
copy-paste everything that is in master for that file